src/status_im/ui/screens/add_new/new_chat/views.cljs

```diff
@@ -57,13 +61,33 @@
 (defn get-validation-label [value]
   (case value
     :invalid
-    (i18n/label :t/user-not-found)
+    (i18n/label :t/profile-not-found)
     :yourself
     (i18n/label :t/can-not-add-yourself)))
 
+(defn search-contacts [filter-text {:keys [name alias nickname]}]
+  (or
+   (string/includes? (string/lower-case (str name)) filter-text)
```

Comment:
Why are you type casting manually `(str name)`? Is there any problem with `(string/lower-case name)`?

Reply:
So I found out that casting serves as a null safety mechanism because `(str nil)` returns an empty string. Removing it causes a regression bug when adding contacts by public key that have not a name or alias. Will re-add it and let me know if there are no other concerns.